cypress/e2e/course_board/createEditDeleteSingleColumnBoard.feature

```diff
@@ -0,0 +1,112 @@
+@regression_test
+@stable_test
+Feature: Course Board - To create, edit and delete a single course board
+
+    As a teacher I want to create, edit and delete a single column course board so that I can use this option.
+
+    Scenario Outline: user creation, course creation, and creating, editing, deleting a single column board in the course
+
+        # pre-condition: teacher and admin log in to create their account in a same school
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        Given I am logged in as a '<student>' at '<namespace>'
+        Given I am logged in as a '<admin>' at '<namespace>'
+
+        # pre-condition: admin creates a course and assign teacher to the course
+        When I go to courses overview
+        When I click on FAB to create a new course depending on sub menu
+        When I enter the course title '<course_name>'
+        When I select course colour as red
+        Then I select teacher '<fullname_teacher>' is selected by default
+        When I click on button Next Steps after entering the course detail in section one
+        When I select the student '<fullname_student>' in the list
+        When I click on button Next Steps after selecting course participant details
+        When I click on button To Course Overview on the finish page
+        # Note: this step is not applicable for the admin user
+        #Then I see the course 'CypressAut Test Creation and Deletion' on the course overview page
+
+        # teacher adds a new Board
```

Comment:
Agree with Ankush, also I would say to mention the type of column board as we have multi-column and single column board
```suggestion
        # teacher adds a new single column board
```